index.html

```diff
@@ -0,0 +1,223 @@
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+
+      <section class="program container" id="program">
+
+        <h2>Main Program</h2>
+        <span class="line"></span>
+
+        <div class="grid">
+        <div class="program-item">
+          <div class="top">
+            <img src="./images/prog_icon_01.png" alt="item1" />
+            <h3>Speaker</h3>
+          </div>
+          <div class="box-content">
+            <p>
+              Be empowered by the words of knowledge and inspiration from our
+              various speakers.
+            </p>
+          </div>
+        </div>
+
+        <div class="program-item">
+          <div class="top">
+            <img src="./images/prog_icon_02.png" alt="item1" />
+            <h3>Showcase</h3>
+          </div>
+
+          <div class="box-content">
+            <p>
+              A great number of african Movies will be presented by different
+              producers across the world.
+            </p>
+          </div>
+        </div>
+
+        <div class="program-item">
+          <div class="top">
+            <img src="./images/prog_icon_03.png" alt="item1" />
+            <h3>The Chat</h3>
+          </div>
+          <div class="box-content">
+            <p>
+              Engage with the speakers and producers on one on one
+              basis.
+            </p>
+          </div>
+        </div>
+
+        <div class="program-item">
+          <div class="top">
+            <img src="./images/prog_icon_04.png" alt="item1" />
+            <h3>Workshop</h3>
+          </div>
+          <div class="box-content">
+            <p>
+              Get engaged in participating in our various projects and ignite
+              your talent.
+            </p>
+          </div>
+        </div>
+
+        <div class="program-item">
+          <div class="top">
+            <img src="./images/prog_icon_05.png" class="top-icon"  alt="item1" />
+            <h3>Connect</h3>
+          </div>
+          <div class="box-content">
+            <p>
+              Meet other movie makers from across Africa and beyond.
+              Connect and network with your teams.
+            </p>
+          </div>
+        </div>
+        </div>
+        
+        <div class='register-btn'>
+          <a href="#" id='join' >Registration for projects</i></a>
+        </div>
+
+      </section>
+      
+
+      <section class='speakers container' id="speaker">
+        <h2>Feature Speakers</h2>
+        <span class="line"></span>
+
+        <div class='speaker-area'>
+        </div>
+      
+      </section>
+
+      
+
+      <section class='partners container' id='footer'>
+        <div class='partners-content'>
+        <h2>Partners</h2>
+        <span class="line"></span>
+
+        <div class='partners-logos'>
+          <img src="./images/mozilla.svg" alt="logo">
+          <img src="./images/google.svg" alt="logo">
```

Comment:
Please define alt attributes with the meaningful description. For here: "Google logo" would be appropriate.